src/shared/components/palette/collection/read-collection-field.html

```diff
@@ -1,14 +1,27 @@
-<table *ngIf="caseField.value && caseField.value.length" class="collection-field-table">
-  <tbody>
-    <tr *ngFor="let item of caseField.value; let i = index">
-      <td>
-        <ccd-field-read [caseField]="{
+<table *ngIf="caseField.value && caseField.value.length " class="collection-field-table">
+  <tbody *ngIf="isDisplayContextParameterAvailable">
+  <tr>
+    <td>
+      <ccd-field-read [caseField]="{
+              id: caseField.label,
+              label: caseField.label,
+              field_type: caseField.field_type.collection_field_type,
+              display_context_parameter: caseField.display_context_parameter,
+              value: caseField.value
+            }" [context]="context"></ccd-field-read>
+    </td>
+  </tr>
+  </tbody>
+  <tbody *ngIf="!isDisplayContextParameterAvailable">
```

Comment:
With ngSwitch we should not need another ngIf.

Reply:
using now